src/js/controller.js

```diff
@@ -107,9 +107,8 @@ export default class Controller extends Component {
             chartConfiguration: payload.indicators[payload.indicatorTitle].chartConfiguration,
             indicatorId: payload.indicatorId
         }
-        if (typeof subindicator.data.originalChildData !== 'undefined' && subindicator.data.originalChildData !== null) {
+        if (typeof subindicator.data.originalChildData !== 'undefined') {
```

Comment:
I find this entire function, along with loadAndDisplayChoropleth hard to understand - any thoughts on how it could be refactored